### PR TITLE
add click handler to increment counter

### PR DESCRIPTION
**Pull request by NoahMarkofski**
Added increment to button

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,12 +1,18 @@
 <html>
   <body>
-    <button onclick="todo()">Count by 5</button>
+    <button id="a" onclick="todo()">Count by 5</button>
```
Comment (deckeraa) [inline index.html]: The ID should be more meaningful than `a`.

Comment (deckeraa) [reply index.html]: Also, if you're attaching the click handler below, you should remove the `onclick="todo()"` in this line.

File: index.html [hunk 2]
```diff
@@ -1,12 +1,18 @@
...
   <script>
     function todo() {
       alert("Please implement a function to increment the counter by the correct amount.")
-    }
+    };
+
+    let count = document.getElementById("counter");
+    
+    document.getElementById('a').onclick = function()  {
+    count.innerHTML = parseInt(count.innerHTML) + 5;
```
Comment (deckeraa) [inline index.html]: This line needs indented.

**Review by deckeraa — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (NoahMarkofski) [reply index.html]: fixed

**Review by deckeraa — APPROVED** [review]

**Pull request merged**